includes/MediaBynderStreamWrapper.inc

```diff
@@ -9,17 +9,17 @@
 class MediaBynderStreamWrapper extends MediaReadOnlyStreamWrapper{
 
     public function interpolateUrl() {
-        $this->base_url = variable_get('media_bynder_url', '');
-
-        if ($parameters = $this->get_parameters()) {
-            return media_bynder_embed_url($parameters['i']);
-        } else {
-            $parameters = media_bynder_uri_parse_style_url($this->uri);
-            if($parameters){
-                return media_bynder_embed_url($parameters['i'], $parameters['style'], $parameters['effects']);
+        $parts = explode('/', $this->uri);
+      
+        foreach ($parts as $part => $part_key) {
+            if ($part_key == 'i'){
+                //idHash of URI
+                if (count($parts) >= ($part + 1)) {
```

Comment:
the URI is supposed to have this format: bynder://f/<id>/i/<idHash>
by splitting (exploding) we extract the idHash and use it
it should always be the one after the /i/ segment. It checks if the array resulting from the split is long enough to store one segment after /i/, if not the fallback is to return FALSE.